lms/static/certificates/sass/_components.scss

```diff
@@ -562,3 +561,83 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: z-index(front);
+    background-color: palette(grayscale, trans); /*dim the background*/
+    vertical-align: middle;
+
+    .badges-modal {
+        color: palette(grayscale, dark);
+        @extend %copy-large;
+        box-sizing: content-box;
+        right: 0;
+        left: 0;
+        max-width: 50%;
+        margin-right: auto;
+        margin-left: auto;
+        position: fixed;
+        top: spacing-vertical(large);
+        background: palette(grayscale, white);
+        z-index: z-index(very-front);
+        border-top: 10px solid palette(primary, light);
+        padding-right: 2em;
+        padding-left: 2em;
+        overflow-x: hidden;
+
+        .close {
+            position: absolute;
+            right: 1em;
+            top: 1em;
+            font-weight: bold;
+            cursor: pointer;
+        }
+
+        .badges-steps {
+            @extend %reset-lists;
+            display: table;
+
+            .step {
+                counter-increment: step-counter;
```

Comment:
NIT: please sync up [property order](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order)